components/filter/filter-dimension-set-date-time-range-value.js

```diff
@@ -104,13 +112,21 @@ class FilterDimensionSetDateTimeRangeValue extends LocalizeCoreElement(LitElemen
 	}
 
 	_getAdditionalContent() {
-		return html`
-			<d2l-input-date-time-range
+		return this.type === 'date'
+			? html`<d2l-input-date-range
+				@change="${this._handleDateChange}"
+				child-labels-hidden
+				end-value="${ifDefined(this.endValue ? getLocalDateTimeFromUTCDateTime(this.endValue) : undefined)}"
+				label="${this.localize('components.filter-dimension-set-date-time-range-value.text')}"
+				label-hidden
+				prefer-fixed-positioning
+				start-value="${ifDefined(this.startValue ? getLocalDateTimeFromUTCDateTime(this.startValue) : undefined)}"
+			></d2l-input-date-range>`
+			: html`<d2l-input-date-time-range
 				@change="${this._handleDateChange}"
 				child-labels-hidden
-				data-dimensionvaluekey="${this.key}"
 				end-value="${ifDefined(this.endValue)}"
-				label="Custom Range"
+				label="${this.localize('components.filter-dimension-set-date-time-range-value.text')}"
```

Comment:
This would (potentially) be the same as the list item text if not set by the consumer. I'll be looking at accessibility in a follow up story so I'll do some testing at that point to see if that's too repetitive.